kernel/src/ipc.rs

```diff
@@ -233,54 +323,35 @@ impl<const NUM_PROCS: usize> LegacyDriver for IPC<NUM_PROCS> {
         &self,
         appid: AppId,
         target_id: usize,
```

Comment:
I can't comment on the lines above this, but the doc comments are now out of date.